building_dialouge_webapp/templates/pages/consumption_result.html

```diff
@@ -1,7 +1,235 @@
-{% extends "base_footer_button.html" %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>consumption result</title>
+    <style>
```

Comment:
The style must be added to our (S)CSS at `building_dialouge_webapp/static/css` instead of the HTML

Reply:
created consumption_results.css, in `building_dialouge_webapp/static/css`  and adjusted styles. Doesn't seem right though.